epilepsy12/tasks.py

```diff
@@ -69,10 +73,114 @@ def async_insert_old_pt_data(csv_path: str = "data.csv"):
 def async_insert_user_data(csv_path: str = "data.csv"):
     insert_user_data(csv_path=csv_path)
 
+
 @shared_task
 def hello():
     """
     THIS IS A SCHEDULED TASK THAT IS CALLED AT 06:00 EVERY DAY
     THE CRON DATE/FREQUENCY IS SET IN SETTING.PY
     """
     logger.debug("0600 cron check task ran successfully")
+
+
+@shared_task
+def download_kpi_summary_as_csv(cohort=6):
+    """
+    Asynchronous task to pull data from KPIAggregation tables and store as dataframe for export as CSV
+    Accepts cohort as optional param, defaults to 6
+    """
+
+    # NATIONAL - SHEET 1
+    # create a dataframe with a row for each measure, and column for each of ["Measure", "Percentage", "Numerator", "Denominator"]
+    # note rows are named ["1. Paediatrician with expertise","2. Epilepsy specialist nurse","3a. Tertiary involvement","3b. Epilepsy surgery referral","4. ECG","5. MRI","6. Assessment of mental health issues","7. Mental health support","8. Sodium valproate","9a. Comprehensive care planning agreement","9b. Comprehensive care planning content","10. School Individual Health Care Plan"]
+
+    NationalKPIAggregation = apps.get_model("epilepsy12", "NationalKPIAggregation")
+
+    national_kpi_aggregation = (
+        NationalKPIAggregation.objects.filter(cohort=cohort)
+        .values(
+            "paediatrician_with_expertise_in_epilepsies_passed",
```

Comment:
@eatyourpeas is there a reason we are specifying each and every attribute here? From my understanding if you call `.values()` without attribute names you get them all, as a dict. We may get a couple we don't actually need, but this is probably acceptable I think. From my POV `national_kpi_aggregation = NationalKPIAggregation.objects.filter(cohort=cohort).values().first()` feels cleaner.

Reply:
Happy to go with this.